src/nnet3/nnet-optimize-utils.cc

```diff
@@ -2839,7 +2839,7 @@ void ComputationExpander::ExpandRowRangesCommand(
       num_rows_new = expanded_computation_->submatrices[s1].num_rows;
   KALDI_ASSERT(static_cast<size_t>(c_in.arg3) <
                computation_.indexes_ranges.size());
-  KALDI_ASSERT(num_rows_old % 2 == 0);
+  //KALDI_ASSERT(num_rows_old % 2 == 0);
```

Comment:
Best to uncomment the line, and if it happens again, point me to where I can debug it.